pkg/operator/ceph/csi/spec.go

```diff
@@ -1106,5 +911,5 @@ func getPrefixFromArg(arg string) (string, bool) {
 }
 
 func EnableCSIOperator() bool {
-	return enableCSIOperator && !IsHolderEnabled()
+	return enableCSIOperator
 }
```

Comment:
@subhamkrai I think it makes sense that holder status should no longer affect CSI operator in 1.16, where holders are fully deprecated. LMK if you foresee any issues with this though.

Reply:
looks good to me